tests/subsys/pm/power_mgmt/testcase.yaml

```diff
@@ -7,4 +7,5 @@ tests:
     integration_platforms:
       - qemu_x86
       - mps2_an385
+      - disco_l475_iot1
```

Comment:
I've answered a bit fast to your question and this platform is not yet updated upstream for PM.
Let's use nucleo_l476rg instead.